### PR TITLE
Chores Rollup

### PR DESCRIPTION
**Pull request by maxarvid**
Link to PT: https://www.pivotaltracker.com/story/show/180309633

### User Story
An and API
In order to be my very best
I would like to have my developers iron out all the little things

### Changes Proposed
- Seed admin account on API
- Use fikas serializer to format dateTime return
- Add content to README
- Add similarity spec in `spec/services`
- Adds serializer spec for fikas

### Screenshots

image

image

### CODE REVIEW

File: app/models/fika.rb [hunk 1]
```diff
@@ -3,6 +3,14 @@ class Fika < ApplicationRecord
   has_and_belongs_to_many :participants, join_table: 'fikas_participants', presence: true
   after_validation :create_calendar_entry
 
+  def formated_start_date
+    start_date.strftime('%-d/%-m %H:%M')
```
Comment (Fraseer) [inline app/models/fika.rb]: Do we not display the year now?

File: app/models/fika.rb [hunk 2]
```diff
@@ -3,6 +3,14 @@ class Fika < ApplicationRecord
   has_and_belongs_to_many :participants, join_table: 'fikas_participants', presence: true
   after_validation :create_calendar_entry
 
+  def formated_start_date
+    start_date.strftime('%-d/%-m %H:%M')    
+  end
+
+  def formated_end_date
+    end_date.strftime('%-d/%-m %H:%M')
```
Comment (Fraseer) [inline app/models/fika.rb]: We dont really use the end date anywhere on the front end, could it be removed? Or does G calendars need it?

File: app/serializers/fikas/index_serializer.rb [hunk 3]
```diff
@@ -1,4 +1,4 @@
 class Fikas::IndexSerializer < ActiveModel::Serializer
-  attributes :id, :start_date, :end_date, :similarity
+  attributes :id, :formated_start_date, :formated_end_date, :similarity
```
Comment (oliverochman) [inline app/serializers/fikas/index_serializer.rb]: why have you decided to change the name to `formated...`?

File: app/models/fika.rb (hunk 1 above)
Comment (maxarvid) [reply app/models/fika.rb]: We've refactored to include year now

File: app/models/fika.rb (hunk 2 above)
Comment (maxarvid) [reply app/models/fika.rb]: This can totally be removed! We've removed it.

File: app/serializers/fikas/index_serializer.rb (hunk 3 above)
Comment (maxarvid) [reply app/serializers/fikas/index_serializer.rb]: Thanks for the steer, we've refactored to override the definition of `:start_date` in the serializer as opposed to defining a new attribute in the model

**Pull request merged**